tests/data_platform/generate_features/test_bedrock_engine.py

```
@@ -0,0 +1,244 @@
+
+        assert result == []
+        assert engine.last_usage == BedrockUsage(0, 0, 0)
+        client.converse.assert_not_called()
+
+    def test_raises_when_response_is_not_json(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+    ) -> None:
+        monkeypatch.setattr(
+            "data_platform.generate_features.engines.bedrock_engine.time.sleep",
+            lambda seconds: None,
```

Ruff (0.16.3)

[warning] Unused lambda argument: `seconds`

(ARG005)